cmd/build

```diff
@@ -87,6 +89,7 @@ elif [ "$1" == check ]; then
     exit 0
 elif [ "$1" == build ]; then
     force=
+    build=build
```

Comment:
I think we should change the name at this point to "build-missing" or something like that.

Reply:
"build" is too generic, yes, and redundant ('cmd/build build"?)  I would go with something like "missing" if that's what sums it up... e.g. "cmd/build missing")

Reply:
Replaced with missing.